plugin/plugins/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.so
+*/main
```

Comment:
This is autogenerated package in directory of every plugin. This is what I have to do so the plugin can be either pre-loaded (statically compiled with go-ipfs, for non-linux builds) or be compiled for dynamic linking as plugin.

Go requires plugins to be defined in package named `main`, and at the same time it doesn't allow for static linking of `main` package. So I have main plugin inside its own package and I generate `main` package that rebinds `Plugins` variable.